dist/redhat/scylla-jmx.spec.mustache

```diff
@@ -41,6 +41,7 @@ fi
 
 %post
 %systemd_post scylla-jmx.service
+/usr/bin/systemctl daemon-reload
 
```

Comment:
There is a file trigger that does it:

```
$ rpm -q --filetriggers systemd
filetriggerpostun scriptlet (using /bin/sh) -- /etc/systemd/system, /usr/lib/systemd/system
if test -f "/var/lib/rpm-state/systemd/needs-reload"; then
    rm -rf "/var/lib/rpm-state/systemd"
    /usr/bin/systemctl daemon-reload
fi
transfiletriggerin scriptlet (using /bin/sh) -- /etc/systemd/system, /usr/lib/systemd/system
# This script will run after any package is initially installed or
# upgraded. We care about the case where a package is initially
# installed, because other cases are covered by the *un scriptlets,
# so sometimes we will reload needlessly.
if test -d /run/systemd/system; then
  /usr/bin/systemctl daemon-reload
fi
transfiletriggerun scriptlet (using /bin/sh) -- /etc/systemd/system, /usr/lib/systemd/system
# On removal, we need to run daemon-reload after any units have been
# removed. %transfiletriggerpostun would be ideal, but it does not get
# executed for some reason.
# On upgrade, we need to run daemon-reload after any new unit files
# have been installed, but before %postun scripts in packages get
# executed. %transfiletriggerun gets the right list of files
# but it is invoked too early (before changes happen).
# %filetriggerpostun happens at the right time, but it fires for
# every package.
# To execute the reload at the right time, we create a state
# file in %transfiletriggerun and execute the daemon-reload in
# the first %filetriggerpostun.

if test -d "/run/systemd/system"; then
    mkdir -p "/var/lib/rpm-state/systemd"
    touch "/var/lib/rpm-state/systemd/needs-reload"
fi
transfiletriggerin scriptlet (using /bin/sh) -- /usr/lib/sysusers.d
# This script will process files installed in /usr/lib/sysusers.d to create
# specified users automatically. The priority is set such that it
# will run before the tmpfiles file trigger.
if test -d /run/systemd/system; then
  /usr/bin/systemd-sysusers || :
fi
transfiletriggerin scriptlet (using /bin/sh) -- /usr/lib/tmpfiles.d
# This script will process files installed in /usr/lib/tmpfiles.d to create
# tmpfiles automatically. The priority is set such that it will run
# after the sysusers file trigger, but before any other triggers.
if test -d /run/systemd/system; then
  /usr/bin/systemd-tmpfiles --create || :
fi
transfiletriggerin scriptlet (using /bin/sh) -- /usr/lib/systemd/catalog
# This script will automatically invoke journal catalog update if files
# have been installed or updated in /usr/lib/systemd/catalog.
if test -d /run/systemd/system; then
  /usr/bin/journalctl --update-catalog || :
fi
transfiletriggerin scriptlet (using /bin/sh) -- /usr/lib/sysctl.d
# This script will automatically apply sysctl rules if files have been
# installed or updated in /usr/lib/sysctl.d.
if test -d /run/systemd/system; then
  /usr/lib/systemd/systemd-sysctl || :
fi
transfiletriggerin scriptlet (using /bin/sh) -- /usr/lib/binfmt.d
# This script will automatically apply binfmt rules if files have been
# installed or updated in /usr/lib/binfmt.d.
if test -d /run/systemd/system; then
  # systemd-binfmt might fail if binfmt_misc kernel module is not loaded
  # during install
  /usr/lib/systemd/systemd-binfmt || :
fi
```

So why wasn't it called?

Reply:
Because there are no file triggers in CentOS 7.